beanlab/vs-code-ai-extension
# 9 Add Prettier and ESLint

After being merged into main, to update local environment, do this:

1. Pull main.
2. Run `npm install`
3. Install the `vscode-eslint` and `prettier-vscode` VS Code extensions.
4. **NOT REQUIRED, BUT RECOMMENDED**: Set prettier as your default formatter and turn on format on save.

### QA Steps

1. Do the steps above.
2. Open the extension with F5.
3. Run the hello world command.
4. The hello world message should display.
